qiskit/qpy/binary_io/circuits.py

```diff
@@ -290,13 +290,17 @@ def _read_instruction(
         gate_class = getattr(quantum_initializer, gate_name)
     elif hasattr(controlflow, gate_name):
         gate_class = getattr(controlflow, gate_name)
+    elif gate_name == "Clifford":
+        pass
```

Comment:
Shouldn't this be?
```suggestion
        gate_class = Clifford
```
Mostly just for consistency. But if we did this then we would be able to drop the special handling in L302 as it would just call `gate_class(*params)`. Or is len(Clifford.params)  > 1 and doing this wouldn't work?

Reply:
Done in [bc1f5e3](https://github.com/Qiskit/qiskit/pull/11495/commits/bc1f5e3d51cb838dc10b9b1f17871a684fa0786a).